src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -22,6 +22,8 @@
 import copy
 from .util import ConfigTests
 
+CONFIG = ConfigTests()
+WS_NAME = CONFIG.get("app_tests", "public_ws_name")
```

Comment:
Had to move these up to globals because wanted to use `WS_NAME` in one of the new global functions (although using `WS_NAME` turned out to be unnecessary)